server/src/inmantals/server.py

```diff
@@ -118,22 +125,83 @@ def flatten(self, line, char):
         assert char < 100000
         return line * 100000 + char
 
+    def create_tmp_project(self) -> str:
+        ten_random_digits = "".join(random.choice(string.digits) for _ in range(10))
+        project_name = "project" + ten_random_digits
+        self.project_dir = os.path.join("/tmp", "vs-code", project_name)
```

Comment:
I think we may have to use the tmp folder of vscode (i.e. on windows, ...) 

I don't know if vscode has any guidelines? 

Otherwise I would use the tempfile of python, which should work on all platforms 
https://docs.python.org/3/library/tempfile.html

Reply:
I'll look into that